src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,19 @@
+.catalog {
+  position: relative;
+  width: $catalogWidth;
+  height: $catalogHeight;
+
+  &__card {
+    position: absolute;
+
+    @for $i from 1 through $columnCount {
+      &:nth-child(#{$columnCount}n + #{$i}) {
+        left: getHorizontalShift($cardWidth, $cardShiftHorizontal, $i);
```

Comment:
it's not a good practice to use positioning to place your cards. 
Just use margins